bin/www

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+const app = require('../app'),
+    debug = require('debug')('nodejs-regular-webapp2:server'),
+    http = require('http'),
+    ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn(),
+    port = normalizePort(process.env.PORT || '3000'),
+    server = http.createServer(app);
+
+app.set('port', port);
+
+app.use('/profile', ensureLoggedIn, require('../profile/profile')());
```

Comment:
czy aby na pewno cała logika wewnątrz /profile/profile jest używana? Mam wrażenie że nigdzie nie ma linka do adresu `/profile` jest natomiast do adresu `user/profile` a tam formularz jest już poprawnie obsłużony.

Upewnij się czy plik profile.js jest potrzebny. jeśli tak to kiedy jest on używany?

Reply:
Wygląda na to, że profile.js nie jest potrzebny. :) Wszystko działa dobrze po jego usunięciu.